tests/unit/backend/models/submission.server.model.spec.ts

```diff
@@ -151,5 +155,83 @@ describe('Submission Model', () => {
         expect(actualWebhookView).toBeNull()
       })
     })
+
+    describe('addWebhookResponse', () => {
+      it('should return updated submission with webhook response and valid submission id', async () => {
```

Comment:
I think the use of the word "with" is a bit misleading here, and what you really mean (and hence should write in the code) is "should return updated submission with webhook response when submission ID is valid"